app/views/admin/users/_form.html.erb

```diff
@@ -210,6 +216,17 @@
     </div>
   </div>
 
+  <% if @user.external_uuids.any? %>
+  <div class="form-group">
+    <%= f.label :trusted_launch_user, 'Trusted Launch User?', class: "col-sm-2 control-label" %>
+    <div class="col-sm-10">
+      <div class="checkbox">
+        <%= f.check_box :trusted_launch_user, checked: true, style: 'margin-left: 0px'  %>
+      </div>
+    </div>
```

Comment:
Accounts doesn't know anything about launches so I'm not super enthused about putting that terminology here (could be in a help text I guess).  Is it clear that unchecking this removed external UUIDs?  Is there another way to do this that is clearer?

Reply:
yeah, I struggled with this.  I didn't want to explicitly say "Remove LMS launch data" since as you say it's not really for LMS only.  

How about about adding some help text that says "Unchecking will remove all links between this account and external systems such as LMS"

Reply:
yes - and make the code more about external uuids

Reply:
can you make the checkbox `:keep_external_uuids` or something?